geode-docs/getting_started/intro_to_clients.html.md.erb

```diff
@@ -0,0 +1,353 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<% set_title("Introduction to", product_name, "Clients") %>
+
+This section provides basic starting points for a variety of <%=vars.product_name %> clients, along
+with very rudimentary connect, put, get, and then a reference to more in-depth docs and examples on
+how to use the client.
+
+## Installing Apache Geode
+
+You can download Apache Geode from the website, run a Docker image, or install with homebrew on OSX.
+
+## Start an Apache Geode Cluster
+
+With a path that contains the bin directory of the installation, start a locator and server.
+
+```
+$ gfsh 
+gfsh> start locator
+gfsh> start server
+```
+
+Create a region
+
+```
+gfsh> create region --name=helloWorld --type=PARTITION 
+```
+
+Shutting Down Apache Geode
+
+```
+gfsh> shutdown --include-locators=true
+```
+
+## Apache Geode Java Client
+
+**Maven Dependencies**
+
+Declare the following dependency and replace $VERSION, with the version of Geode that you have installed.
+
+```
+<dependencies>
+     <dependency>
+        <groupId>org.apache.geode</groupId>
+        <artifactId>geode-core</artifactId>
+        <version>$VERSION</version>
+    </dependency>
+</dependencies>
+```
+
+**Gradle Dependencies**
+
+``
+dependencies {
+  compile "org.apache.geode:geode-core:$VERSION"
+}
+``
+
+Connect to Geode from your application
+ 
+```
+import org.apache.geode.cache.client.ClientCache;
+import org.apache.geode.cache.client.ClientCacheFactory;
+
+public class HelloWorld {
+
+ public static void main(String[] args) {
+  ClientCache cache = new ClientCacheFactory().addPoolLocator("127.0.0.1", 10334).create();
+
+   System.out.println(cache.getDefaultPool().getLocators());
```

Comment:
Fix indentation here and in line 87

Reply:
done

Reply:
Line 87 (which is line 92 now after your new commits) is still bad indented:
```
public static void main(String[] args) {
    ClientCache cache = new ClientCacheFactory().addPoolLocator("127.0.0.1", 10334).create();

    System.out.println(cache.getDefaultPool().getLocators());

  cache.close();
 }
```

Reply:
done.

Reply:
Sorry, my fault: I added the code there to illustrate the bad indentation, but not to be included as it is... The `cache.close()` line should be aligned with the previous one. So the complete code snippet should look like:

```
public static void main(String[] args) {
    ClientCache cache = new ClientCacheFactory().addPoolLocator("127.0.0.1", 10334).create();

    System.out.println(cache.getDefaultPool().getLocators());

    cache.close();
 }
```

Reply:
I wondered about that. Thanks for the clarification -- fixed in source file.